api/controllers/teams.controller.ts

```diff
@@ -76,7 +85,7 @@ class TeamsController {
       if (!teamMember) {
```

Comment:
TeamMembersServices should be the one throwing an error if a team member is not found. 
When we do deleteTeamMemberFromTeam, we need to encapsulate in a try catch block. If Team member is not found, it will throw an error, and the catch will capture the error.